ORGANISATIONS.yaml

```
@@ -66,3 +66,15 @@ ifb:
     url: https://www.france-bioinformatique.fr/
     avatar: "/training-material/shared/images/ifb.png"
     github: false
+
+Australian BioCommons:
```

```suggestion
Australian-BioCommons:
```

we don't lint on it, but I suspect spaces can cause problems for some users, so, if it's ok, having a `-` in the identifier could be slightly safer.

---

Thanks. I've updated this.